middleware_jwt_test.go

```diff
@@ -179,10 +179,10 @@ var jwtWithCentralDefNoPolicyBaseField = `
 
 `
 
-const JWTSECRET = "9879879878787878"
+const jwtSecret = "9879879878787878"
 
 // openssl genrsa -out app.rsa
-const JWTRSA_PRIVKEY = `
+const jwtRSAPrivKey = `
```

Comment:
It's for readability, I find it useful to know that "Someone else has handled this for me". The go http lib provides status code constants, which is great, maybe it should be a package level thing, they are all prefixed by 'status'.